var/ramble/repos/builtin/applications/gromacs/application.py

```diff
@@ -147,26 +147,28 @@ class Gromacs(SpackApplication):
                       description='Input path for JCP_benchmark {workload_name}',
                       workloads=['ion_channel', 'rnase_cubic'])
 
-    figure_of_merit('Core Time', log_file='{experiment_run_dir}/md.log',
+    log_str = '{experiment_run_dir}/md.log'
```

Comment:
This should probably use `os.path.join` to make the path portable.

Reply:
Done